components/LoginScreen.jsx

```diff
@@ -0,0 +1,156 @@
+import { StatusBar } from "expo-status-bar";
+import { useEffect, useState } from "react";
+import { FontAwesome5 } from "@expo/vector-icons";
+import {
+  KeyboardAvoidingView,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+  Image,
+} from "react-native";
+
+import {
+  getAuth,
+  createUserWithEmailAndPassword,
+  signInWithEmailAndPassword,
+} from "firebase/auth";
+import { auth } from "../firebaseConfig";
+import { useNavigation } from "@react-navigation/native";
+
+export default LoginScreen = ({ navigation }) => {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const navigation1 = useNavigation();
+
+  useEffect(() => {
+    const unsubscribe = auth.onAuthStateChanged((user) => {
+      if (user) {
+        navigation1.navigate("Home");
+      }
+    });
+
+    return unsubscribe;
+  }, []);
+
+  const handleSignUp = () => {
+    createUserWithEmailAndPassword(auth, email, password)
+      .then((userCredentials) => {
+        const user = userCredentials.user;
+        console.log("Registered with", user.email);
+        alert("Account Registered");
+      })
+      .catch((error) => {
+        alert(error.message);
+      });
+  };
+
+  const handleLogin = () => {
+    signInWithEmailAndPassword(auth, email, password)
+      .then((userCredentials) => {
+        const user = userCredentials.user;
+        console.log("Logged in with", user.email);
+        alert("Logged in");
+      })
+      .catch((error) => {
+        alert(error.message);
```

Comment:
Not for this ticket but we can create another ticket here for later in the sprint to add more specific error handling. Ie if a user does not exist, prompt the user to sign up instead